pkg/image/registry/imagestream/strategy_test.go

```diff
@@ -312,6 +312,7 @@ func TestTagVerifier(t *testing.T) {
 			}
 			expectedSar := &authorizationapi.SubjectAccessReview{
 				Action: authorizationapi.Action{
+					Group:        "image.openshift.io",
```

Comment:
I think we have to wait at least one release with legacy groups because I am not sure how this would behave during a rolling update with multiple API servers.